sklearn/feature_selection/tests/test_sequential.py

```diff
@@ -13,28 +13,17 @@
 from sklearn.cluster import KMeans
 
 
-@pytest.mark.parametrize("n_features_to_select", (0, 5, 0.0, -1, 1.1))
-def test_bad_n_features_to_select(n_features_to_select):
-    X, y = make_regression(n_features=5)
-    sfs = SequentialFeatureSelector(
-        LinearRegression(), n_features_to_select=n_features_to_select
-    )
-    with pytest.raises(ValueError, match="must be either 'auto'"):
-        sfs.fit(X, y)
-
-
-def test_bad_direction():
-    X, y = make_regression(n_features=5)
-    sfs = SequentialFeatureSelector(
-        LinearRegression(), n_features_to_select="auto", direction="bad"
-    )
-    with pytest.raises(ValueError, match="must be either 'forward' or"):
+def test_bad_n_features_to_select():
+    n_features = 5
+    X, y = make_regression(n_features=n_features)
+    sfs = SequentialFeatureSelector(LinearRegression(), n_features_to_select=n_features)
+    with pytest.raises(ValueError):
```

Comment:
It could be better to match the error message by adding a substring to match:

```python
err_msg = "n_features_to_select must be either"
with pytest.raises(ValueError, match="xxx"):
    sfs.fit(X, y)
```